src/toady/pr_selection.py

```
@@ -0,0 +1,222 @@
+        if len(pull_requests) == 1:
+            return self._handle_single_pr(pull_requests[0])
+
+        # Scenario 3: Multiple open PRs
+        if allow_multiple:
+            return self._handle_multiple_prs(pull_requests)
+        else:
+            # If not allowing multiple selection, treat as error
+            raise PRSelectionError(
+                f"Found {len(pull_requests)} open pull requests, but multiple "
+                "selection is not allowed in this context"
+            )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)